### DISCUSSION TITLE
Set max padding?

### DISCUSSION THREAD

**Discussion by hverlin** [firecow/gitlab-ci-local#839] [Ideas]
Hi, 

Thanks for this great project! 

I was just wondering if it would be possible to update the logic that adds a padding for the job name here:
https://github.com/firecow/gitlab-ci-local/blob/5062d6a638fffa3f7c8fb322df463399d32d5015/src/parser.ts#L163-L167

The reason is that for some parallel jobs, the name can become very long, and it's not possible to control it as they are generated.
https://docs.gitlab.com/ee/ci/yaml/#parallelmatrix
> parallel:matrix jobs add the variable values to the job names to differentiate the jobs from each other
Moreover, the current logic takes into account all jobs (not just the ones that will get executed)

Could we add a yargs option or environment variable to simply set `this._jobNamePad` to 0?
Happy to raise a PR

**Comment by firecow** [comment]
Yes, just close this discussion and create and feature issue and start working on a PR

`--max-jobname-lenght=100`

**Reply by hverlin** [reply]
Thanks for the prompt answer, here is the PR: https://github.com/firecow/gitlab-ci-local/pull/841
